profile.php

```diff
@@ -92,6 +92,7 @@
     return helper::reason_display((int)$render($text));
 };
 
+$data['datasize'] = round($profile->datasize / 1000, 1) . ' kB';
```

Comment:
I believe you might be able to use `display_size()` from `lib/moodlelib.php` for this purpose, so you aren't managing the size conversions yourself.